endpoints/events/event_test.go

```diff
@@ -663,3 +696,44 @@ func TestShouldParseEventCorrectly(t *testing.T) {
 	assert.Equal(t, 0, len(errs))
 	assert.EqualValues(t, expected, er)
 }
+
+func TestEventRequestToUrl(t *testing.T) {
```

Comment:
This looks good and you've got the code coverage. My personal preference would be to add the following scenarios by either altering the existing cases or adding new ones for completeness but I'm nitpicking 🙂:
- `request.Timestamp <= 0`
- `request.Bidder == ""`
- `opt == ""`

Up to you if you want to add.

Reply:
Happy to do so.